core/src/main/java/org/elasticsearch/cli/EnvironmentAwareCommand.java

```diff
@@ -70,13 +67,9 @@ protected void execute(Terminal terminal, OptionSet options) throws Exception {
         putSystemPropertyIfSettingIsMissing(settings, "path.home", "es.path.home");
         putSystemPropertyIfSettingIsMissing(settings, "path.logs", "es.path.logs");
 
-        final String pathConf = pathConfOption.value(options);
-        execute(terminal, options, createEnv(terminal, settings, getConfigPath(pathConf)));
-    }
+        final Path pathConf = Paths.get(System.getProperty("es.path.conf"));
```

Comment:
Should there be a clearer error if the property wasn't set?
While that shouldn't happen (since all our scripts explicitly set it), if it's missing then you get a not-very-helpful NPE.